appendices/special-characters/text.md

```diff
@@ -1,21 +1,23 @@
 # Text
 
-## **Syntax**
+## Syntax
 
-```text
+```G1ANT
 ‴The text to use or save‴
 ```
 
-## **Description**
+## Description
 
 Text special characters `‴‴` are required to declare text value within all available commands. This text can be used for displaying messages, storing file paths containing spaces, as an input in documents or dialog boxes etc., and can also be assigned to variables.
```

Comment:
can you modify this text a little? just remove "required" word because ‴‴ are only required when your text value contains some spaces

Reply:
```suggestion
Text special characters `‴‴` are required to declare text value containing spaces for arguments in all available commands. This text can be used for displaying messages, storing file paths, as an input in documents or dialog boxes etc., and can also be assigned to variables.
```